plugins/history_plugin/history_plugin.cpp

```diff
@@ -142,6 +148,8 @@ namespace eosio {
          std::set<filter_entry> filter_out;
          chain_plugin*          chain_plug = nullptr;
          fc::optional<scoped_connection> applied_transaction_connection;
+         fc::microseconds history_api_timeout_us = fc::microseconds(100000);
+         int64_t last_id_pruned = 0;
```

Comment:
As you have it, this is not last id pruned but rather next_id_to_prune. Either rename to something more descriptive or update to value before remove so that it is actually last_id_pruned.